caddyconfig/httpcaddyfile/builtins_test.go

```diff
@@ -213,3 +214,45 @@ func TestRedirDirectiveSyntax(t *testing.T) {
 		}
 	}
 }
+
+func TestErrorLine(t *testing.T) {
+	for i, tc := range []struct {
+		input     string
+		errorFunc func(err error) bool
+	}{
+		{
+			input: `(t1) {
+					abort {args.:}
+				}
+				:8080 {
+					import t1
+					import t1 true
+				}`,
+			errorFunc: func(err error) bool {
+				return err != nil && strings.Contains(err.Error(), "Caddyfile:6(import t1):2")
+			},
+		},
+		{
+			input: `(t1) {
```

Comment:
Test invalid import

Reply:
Explained in https://github.com/caddyserver/caddy/pull/5249#discussion_r1106925556, this second test just makes sure the error line is correct by being at least different than the other test.